client/ayon_maya/api/pipeline.py

```diff
@@ -347,11 +347,18 @@ def parse_container(container):
         container (str): A container node name.
 
     Returns:
-        dict: The container schema data for this container node.
+        dict[str, Any]: The container schema data for this container node.
 
     """
     data = lib.read(container)
 
+    required = ["id", "name", "namespace", "loader", "representation"]
+    missing = [key for key in required if key not in data]
+    if missing:
+        log.warning("Container '%s' is missing required keys: %s",
+                    container, missing)
+        return {}
```

Comment:
Shoulnd't this return `None` rather than empty dictionary?

Reply:
Not according to the functions type hint :D
Checked the usage of the function and noticed this may also influence `setdress` in maya - so I may want to do this differently.

Reply:
Then we should change type hint? I meant returning empty dictionary if there are missing keys is worse than returning whatever is stored in scene.

Reply:
Implemented with https://github.com/ynput/ayon-maya/pull/179/commits/5d689b57cf45fda4d89490d41ca6fa1c8250b482